Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -646,558 +587,373 @@ protected virtual void Dispose(bool disposing)
             }
         }
 
-        /// <summary>
-        /// Tries to identify the string and return an object of that class.
-        /// </summary>
-        /// <param name="addr">String to parse.</param>
-        /// <param name="result">IPObject to return.</param>
-        /// <returns><c>true</c> if the value parsed successfully, <c>false</c> otherwise.</returns>
-        private static bool TryParse(string addr, out IPObject result)
+        /// <inheritdoc/>
+        public bool TryParseInterface(string intf, [NotNullWhen(true)] out IReadOnlyList<IPData>? result)
         {
-            if (!string.IsNullOrEmpty(addr))
+            var resultList = new List<IPData>();
+            if (string.IsNullOrEmpty(intf) || _interfaces is null)
             {
-                // Is it an IP address
-                if (IPNetAddress.TryParse(addr, out IPNetAddress nw))
-                {
-                    result = nw;
-                    return true;
-                }
+                result = resultList.AsReadOnly();
```

Comment:
Since it is nullable, you could just return null here or at the very least use Array.Empty to avoid the allocation?